lib/nimhcr.nim

```diff
@@ -423,7 +423,7 @@ when defined(createNimHcr):
     if modules.contains(name):
       unloadDll(name)
     else:
-      modules.add(name, newModuleDesc())
+      modules[name] = newModuleDesc()
```

Comment:
tables warnings

Reply:
this changes semantics unless name is a unique key; can you double check this for correctness?

also the doc comment in tables.add is misleading:
```nim
proc add*[A, B](t: var Table[A, B], key: A, val: sink B) {.deprecated:
    "Deprecated since v1.4; it was more confusing than useful, use `[]=`".} =
```
given that this changes semantics

Reply:
it should the same semantics in this situation, `if modules.contains(name):` make sure the name is unique.